config/prow/cluster/monitoring/mixins/lib/config.libsonnet

```diff
@@ -68,7 +68,9 @@
       {instance: "104.197.27.114:9090", type: "node-e2e-project", friendly: "Node e2e project"},
       {instance: "104.197.27.114:9090", type: "scalability-project", friendly: "Scalability project"},
       {instance: "35.225.208.117:9090", type: "scalability-project", friendly: "Scalability project (k8s-infra)"},
-      {instance: "104.197.27.114:9090", type: "scalability-presubmit-project", friendly: "Scalability presubmit project"}
+      {instance: "104.197.27.114:9090", type: "scalability-presubmit-project", friendly: "Scalability presubmit project"},
+      {instance: "TODO:9090", type: "image-builder-aws-account", friendly: "Image Builder - AWS"},
```

Comment:
Maybe @dims could provide guidance on which boskos url we should target for this

Reply:
better for @spiffxp ( or @bartsmykla ) to help with this. I am not quite sure how this was done before.

Reply:
Unfortunately I don't have the knowledge about it either :-(

Reply:
If I had to guess one is related to the old google managed infrastructure and the other is related to the new wg-k8s-infra managed infrastructure, but I'm not sure as to where things are in the migration process

Reply:
Hopefully this helps:

https://github.com/kubernetes/test-infra/blob/0d30bea7effaff7934b0ebce2e1c0b6335b125ba/config/prow/cluster/monitoring/additional-scrape-configs_secret.yaml#L13-L15

Reply:
but given that you're adding these to the same boskos config that contains resource types such as `aws-account` and `node-e2e-project`, I think you want `104.197.27.114` here.

Reply:
Thanks for digging it up @ixdy